types/expo/index.d.ts

```diff
@@ -2561,13 +2699,14 @@ export class Svg extends Component<{ width: number, height: number, viewBox?: st
  * Take Snapshot
  */
 export function takeSnapshotAsync(
-    view?: (number | React.ReactElement<any>),
+    node: number | React.ReactElement<any> | React.RefObject<any>,
     options?: {
-        width?: number,
-        height?: number,
-        format?: 'png' | 'jpg' | 'jpeg' | 'webm',
-        quality?: number,
-        result?: 'file' | 'base64' | 'data-uri',
+        width?: number;
+        height?: number;
+        format: 'png' | 'jpg' | 'raw' | 'webm';
```

Comment:
`raw` is not listed in [the documentation](https://docs.expo.io/versions/latest/sdk/take-snapshot-async/#takesnapshotasyncview-options)?

Reply:
https://github.com/expo/expo/pull/2771/files#diff-e1bb088e01a152f522c27cac3147cb34L9

Reply:
Thanks for the link.